heaplayers.h

```diff
@@ -38,6 +38,11 @@ namespace HL {}
 
 // #define HL_EXECUTABLE_HEAP 1
 
+// Define HL_NAMED_HEAP as 1 to id an address range (on Linux)
+
+// #define HL_NAMED_HEAP 1
+#define HL_HEAP_NAME "Heap Layer"
```

Comment:
Change as follows (add 's' to Heap Layer, and make it easy to substitute in a build of a client like Hoard):

```
#if !defined(HL_HEAP_NAME)
#define HL_HEAP_NAME "Heap Layers"
#endif
```